ci/release.js

```diff
@@ -95,30 +95,22 @@ async function main() {
   )
 
   console.log("--- Committing release changes ---")
-
-  const VoyagerCommit = execSync("git rev-parse --short=6 HEAD")
-    .toString()
-    .trim()
-
-  console.log("Voyager commit:", VoyagerCommit)
```

Comment:
The idea behind fetching and posting the commit was to enable reproducability of the build. `HEAD` will change over time. Do you think this is not needed? Is there another way to post the commit?